metadata-ingestion/src/datahub/ingestion/source/powerbi/powerbi-lexical-grammar.rule

```diff
@@ -304,6 +305,7 @@ unary_expression:   type_expression
                 |   "+" unary_expression
                 |   "_" unary_expression
                 |   "not" unary_expression
+                |   expression
```

Comment:
nice - so this broadens the definition so that unary expression can be any type of expression?

Reply:
yep - this isn't strictly correct because it can cause incorrect order of operations in the AST (e.g. addition before multiplication or something), but we don't really care about that here